stingray/modeling/posterior.py

```diff
@@ -28,6 +28,43 @@
 
 logmin = -10000000000000000.0
 
+
+def assign_if_not_finite(value, default):
```

Comment:
Why is this a function in `posterior.py`. It seems to me like this should be in `utils.py` somewhere? Also, don't we already have a function like this somewhere? I feel like we must :)